packages/shared/forks/lowPriorityWarningWithoutStack.www.js

```diff
@@ -6,4 +6,9 @@
  */
 
 // This "lowPriorityWarning" is an external module
-export default require('lowPriorityWarning');
+const lowPriorityWarning = require('lowPriorityWarning');
+
+export default function(...args) {
+  // eslint-disable-next-line react-internal/warning-args, react-internal/no-production-logging
+  return lowPriorityWarning(false, args);
```

Comment:
This one is interesting. Did we change this API in this PR? If not, what's the www API? Has this been wrong the whole time?

Reply:
actually ignore me, we did change its API